XSLT/tslaqdctomods.xsl

```diff
@@ -80,5 +92,33 @@
       <url usage="primary" access="object in context"><xsl:apply-templates/></url>
       <url access="preview"><xsl:value-of select="$identifier-preview-url"/></url>
     </xsl:template>
-  
+    
+    <!-- creator(s) -->
+    <xsl:template match="dc:creator">
+        <xsl:variable name="creator-tokens" select="tokenize(., ';')"/>
+        <xsl:for-each select="$creator-tokens">
+            <name>
+                <namePart><xsl:value-of select="normalize-space(.)"/></namePart>
+                <role>
+                    <roleTerm authority="marcrelator" authorityURI="http://id.loc.gov/vocabulary/relators/cre">Creator</roleTerm>
+                </role>
+            </name>
+        </xsl:for-each>
+    </xsl:template>
+    
+    <!-- description -->
+    <xsl:template match="dc:description">
+        <abstract><xsl:apply-templates/></abstract>
+    </xsl:template>
+    
+    <!-- date -->
+    <xsl:template match="dc:date">
```

Comment:
Are we just worried about edm:TimeSpan/dc:date (not edm:TimeSpan/edm:begin)?  If so, this is good.

Reply:
That's all I was initially worried about, but should we consider adding the edm:TimeSpan/edm:begin in the future?